nearlib/tsapi/near.ts

```diff
@@ -0,0 +1,306 @@
+type BufferTypeIndex = u32;
```

Comment:
this is outdated (doesn't have docs).

the one in templates has docs https://github.com/nearprotocol/NEARStudio/blob/master/templates/todomvc/assembly/near.ts

also let's not put this into nearlib. It's gonna live in it's own package once we figure out solution for https://github.com/AssemblyScript/assemblyscript/issues/448